4-esqueleto/README.md

```diff
@@ -1,3 +1,76 @@
 # Plano de esqueleto
 
-## Referencias
\ No newline at end of file
+## Wireframes de las páginas de Aupa
+
+### Introducción (Lisandro)
+
+En este documento se ha de desarrollar el plano esqueleto, para ello, primero se detectan y definen las páginas del sitio web Aupa, luego se realiza un wireframe por cada una y por último abordando cada una de las páginas con la perspectiva de los tres tipos de diseños:
+
+- Diseño de la interfaz
+- Diseño de la navegación
+- Diseño de la información
+
+A continuación se muestra el resultado de este proceso.
+
+### Página principal (Angie)
+
+### Buscador (Estelle)
+
+![Buscador](buscador.png)
+
+### Contacto (Estelle)
+
+![Contacto](contactos.png)
+
+### Foro 
+![IMG1](/4-esqueleto/Wireframes.jpeg)
+
+En la segunda imagen, podemos ver la pagina principal de la pestaña “foro”. 
+Para la interfaz, será una interfaz muy simple y para la navegación, será muy fácil, podremos ver los sujetos del foro, el numero de vistas y de posts serán visibles. 
+En la primera, es la pagina que los usuarios podrán ver cuando van a hacer click en un sujeto. La navegación va a ser muy simple como podremos ver en la imagen.
+
+### Páginas de similar diseño (Lisandro)
+ 
+ Wireframe con diferentes niveles de detalle:
+
+ - **Wireframe 1**
+
+ ![Wireframe 1](https://github.com/DeustoPWEB2018/proyectoweb-migraciones/blob/3264517cc160c33049ec5854babaebc6d5aba64c/4-esqueleto/4_Wireframe_pag_similares_1.jpg)
+
+ - **Wireframe 2**
+
+ ![Wireframe 2](https://github.com/DeustoPWEB2018/proyectoweb-migraciones/blob/3264517cc160c33049ec5854babaebc6d5aba64c/4-esqueleto/4_Wireframe_pag_similares_2.jpg)
+
+ - **Wireframe 3**
+ 
+ ![Wireframe 3](https://github.com/DeustoPWEB2018/proyectoweb-migraciones/blob/3264517cc160c33049ec5854babaebc6d5aba64c/4-esqueleto/4_Wireframe_pag_similares_3.jpg)
+
+
+  **Diseño de la interfaz**
+
+ Hay dos ejes importantes:
+
+ *Simplicidad:* Tiene que entenderse fácilmente con un simple vistazo. Se debe contribuir en el diseño de la interfaz para que tanto los usuarios habituales como los nuevos puedan sentirse familiarizados con la página, la distribución de los elementos y las acciones que se pueden hacer de forma rápida para llegar al contenido que están buscando de la forma mas practica posible.
+
+ *Coherencia:*  El sitio tiene un orden jerárquico en una relación padre-hijo entre las distintas páginas. Dado esto, la interfaz debe transmitir una comprensión clara de estas relaciones para aumentar la efectividad del usuario dentro de las páginas.
+
+ Con la idea de anticipar los movimientos posibles de los usuarios dentro de un articulo, se deja en la parte superior el menú con las pestañas principales del sitio. 
+
+ La calidad del diseño está dispuesta para mostrar una estética y diseño estándar para todos los artículos dentro de cada pestaña, esto ayuda en el proceso de familiarización con el sitio que experimenta el usuario.
+
+  **Diseño de la navegación**
```

Comment:
Otra vez, solamente habláis a nivel teórico. La navegación global se recoge en la barra de navegación de la parte superior, que ofrece enlaces a las secciones principales del sitio. Pero tenéis también otros tipos de navegación, a través del buscador o de los enlaces de migas de pan (`AUPA > Documentos > Demanda de asilo`)... ¿Qué podéis decir de cada uno de ellos? ¿Qué necesidades del usuario cubre cada uno?

Reply:
Ok Miren, vamos a mejorar el abordaje de este punto. 
Chicas yo voy a introducir los cambios.

Reply:
Resuelto!